tests/core/providers/test_wsv2_provider.py

```diff
@@ -90,11 +92,12 @@ async def test_async_make_request_returns_cached_response_with_no_recv_if_cached
     desired_response = {"jsonrpc": "2.0", "id": 0, "result": "0x1337"}
     provider._request_processor.cache_raw_response(desired_response)
 
+    # request with id `0` should return the cached response
     response = await method_under_test(RPCEndpoint("some_method"), ["desired_params"])
     assert response == desired_response
 
     assert len(provider._request_processor._request_response_cache) == 0
-    assert not provider._ws.recv.called  # type: ignore
+    assert provider._ws.recv.not_called  # type: ignore
```

Comment:
I tried a few lint runs without the `type: ignore` and it seems to be unnecessary. Does it work for you if you remove it?

Reply:
Oh good to know. This may have just been a PyCharm warning then. I'll pull it, thanks 👍🏼